dev_guide/compute_resources.adoc

```diff
@@ -331,38 +331,40 @@ implemented. There is no resource quota or limit range support for these
 resources, and they have no impact on QoS.
 ====
 
-Opaque integer resources are resources that begin with the prefix
-`pod.alpha.kubernetes.io/opaque-int-resource-`. The API server
+Opaque integer resources are called _opaque_ because {product-title} 
+does not know what the resource is, but will schedule a pod on a node 
+only if enough of that resource is available. They are called _integer resources_ 
+because they must be available, or _advertised_, in integer amounts. The API server
 restricts quantities of these resources to whole numbers. Examples of
-_valid_ quantities are `3`, `3000m` and `3Ki`. Examples of _invalid_
-quantities are `0.5` and `1500m`.
+_valid_ quantities are `3`, `3000m` and `3Ki`. 
 
-There are two steps required to use opaque integer resources. First, the cluster
-operator must advertise a per-node opaque resource on one or more nodes. Second,
-users must request the opaque resource in pods.
+Some example use cases for opaque integer resources:
 
-To advertise a new opaque integer resource, the cluster operator should
-submit a `PATCH` HTTP request to the API server to specify the available
-quantity in the `status.capacity` for a node in the cluster. After this
-operation, the node's `status.capacity` will include a new resource. The
-`status.allocatable` field is updated automatically with the new resource
-asynchronously by the kubelet.
+* LLC (last-level cache) allocation.
+* GPU devices
+* FPGA devices
+* MICs (Many-Integrated Core)
+* Slots for sharing bandwidth to a parallel file system.
+// https://github.com/kubernetes/kubernetes/issues/28312
```

Comment:
I can't tell how the above are use cases. Is there more information that can be put in? They just seem like a list of components, or something similar. But also, I'm not sure what the other acronyms are. This might need to be elaborated on.

Reply:
@bfallonf 
I took that list from a Kubernetes issue in GithHub:
https://github.com/kubernetes/kubernetes/issues/28312

Which was reproduced in this article. The author expanded on the list a bit (but I didn't want to take his words). 
http://www.devoperandi.com/kubernetes-not-just-cpu-and-ram-opaque-integer-resources/

I can pull the list, if you think it inappropriate or not helpful.

Reply:
@mburke5678 I think the wording is better.